### PR TITLE
Update to v0.3.0

### PR DESCRIPTION
**Pull request by ttmarek**

### CODE REVIEW

**Comment by sudo-suhas** [comment]
Hey @ttmarek, correct me if I am wrong, the changes from `0.2.1` to `0.3.0` are backward compatible right?
In `0.x.y` version range, semver rules are slightly different from `x.y.z` and backward compatible changes should result in incrementing `y`(not `x`). Here's some relevant documentation - https://github.com/dbrock/semver-howto/tree/06b257a43ab9caa5b523d6fb935a82c644e424b4#alpha-and-beta-0xy.

And some code:
```
λ node
> const semver = require('semver')
undefined
> semver.satisfies('0.3.0', '^0.2.0')
false
> semver.satisfies('1.3.0', '^1.2.0')
true
> semver.satisfies('0.2.1', '^0.2.0')
```

Basically, if I have declared a dependency for `consolemock` as `^0.2.0`, it won't get `^0.3.0` because it is considered a breaking change in the `0.x.y` version series.

Also I noticed that no tag was created for the release. Git tags are considered good practice and [`npm version`](https://docs.npmjs.com/cli/version) does this for you automatically.

**Comment by ttmarek** [comment]
> Hey @ttmarek, correct me if I am wrong, the changes from 0.2.1 to 0.3.0 are backward compatible right?

Yes, that's true.

> In 0.x.y version range, semver rules are slightly different from x.y.z and backward compatible changes should result in incrementing y(not x).

To be honest, I didn't know that was in the semver standard. I've been following the usual Major Minor Patch that's standard post v1.0.0. Thank you :slightly_smiling_face:. To think of it, I might bump the next release up to v1.0.0. I don't think the API will change much, and it'll make little confusions like these less likely.

> Also I noticed that no tag was created for the release. Git tags are considered good practice and npm version does this for you automatically.

Yes, I know. This started out as a project to help me on another OSS project. I didn't think anyone else would use it. I will be adding tags and releases from now on. :+1:
